check_line_length.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import re
+import sys
+
+LIMIT = 75
+
+
+def Find(string):
+    """
+    Check if there is a url present in the string
+    """
+
+    url = \
+        re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+                   , string)
+    if url:
+        return True
+    else:
+        return False
+
+
+def limit_line_length(file_path):
+    """
+    Check if the line is in the given limit or not
+    """
+
+    file = file_path.split('/')[-1]
+
+    with open(file_path) as f:
+        lines = f.readlines()
+
+    for (line_number, line) in enumerate(lines, start=1):
+        length = len(line)
+        if length > LIMIT and Find(line) is not True:
+            print('line {} in file {} is longer than {}'.format(line_number,
```

Comment:
Can you add the word `characters` at the end of this message ? Just to complete the sentence. :smile: